lib/Horde/Db/Adapter/Postgresql/Schema.php

```diff
@@ -1143,9 +1155,8 @@ public function postgresqlVersion()
     {
         if (!$this->_version) {
             try {
-                $version = $this->selectValue('SELECT version()');
-                if (preg_match('/PostgreSQL (\d+)\.(\d+)\.(\d+)/', $version, $matches))
-                    $this->_version = ($matches[1] * 10000) + ($matches[2] * 100) + $matches[3];
+                $version = $this->selectValue('SHOW server_version_num');
```

Comment:
server_version_num is only available with psql 9.4, but we still support earlier versions. Please remove this part of the patch.

Reply:
Probably you've to dig deeper in pg documentation.

https://www.postgresql.org/docs/8.2/runtime-config-preset.html#GUC-SERVER-VERSION-NUM

But the reference manual start to link to that page from the System information functions just from 9.5 version

https://www.postgresql.org/docs/9.5/functions-info.html

But here

https://www.postgresql.org/docs/8.2/config-setting.html

«The SHOW command allows inspection of the current values of all parameters.»

8.2 is waaaaay to old and last version that is supported by the pg community is 9.4.
It's a bit hard to actually check since it is hard to find a running version of such an old pg or install one (I wonder if it still compile on a "modern" system).

If you want to be extra sure you can actually access server_version_num I guess your best bet is to contact some historian in the pg community.